.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
   matrix:
   - PYTHON_VERSION=3.6 DMOJ_USE_SECCOMP="yes"
   - PYTHON_VERSION=3.7 DMOJ_USE_SECCOMP="yes"
-  - PYTHON_VERSION=3.6 DMOJ_USE_SECCOMP="no"
+  - PYTHON_VERSION=3.8 DMOJ_USE_SECCOMP="yes"
```

Comment:
The fact that this is out of order hurts

Reply:
I feel like seccomp grouping > version grouping. Do you want it to be re-ordered?